### ISSUE TITLE
Needs Additions: To-Do Lists

### ISSUE THREAD

**Issue by fable-a** [fable-a/System-Library#16]
Needs accessibility updates with drag and drop reordering, better delete button visiblity, automatic reordering, empty list items when clicked out of should delete themselves, should be able to change the title through clicking on the title in the top bar.
